x-pack/plugins/siem/server/utils/build_validation/route_validation.ts

```diff
@@ -32,8 +33,9 @@ export const buildRouteValidation = <T extends rt.Mixed, A = rt.TypeOf<T>>(
 ) =>
   pipe(
     schema.decode(inputValue),
+    decoded => exactCheck(inputValue, decoded),
     fold<rt.Errors, A, RequestValidationResult<A>>(
-      (errors: rt.Errors) => validationResult.badRequest(failure(errors).join('\n')),
+      (errors: rt.Errors) => validationResult.badRequest(formatErrors(errors).join(',')),
```

Comment:
NOTE: This is where I changed the `\n` to be `,` to be consistent with down stream response checks and tests. I am open to what we want to do here.

Notice the `formatErrors` I use which is the same as other projects within Kibana which formats the default io-ts from something to more intelligible. I am open to how we want to change the error reporting from io-ts back to the front end.

Reply:
nit: `','` is the default separator for `Array.join`